### DISCUSSION TITLE
usb_claim_interface error -6

### DISCUSSION THREAD

**Discussion by YarinT** [rtl-airband/RTLSDR-Airband#322] [Q&A]
Hello, I’m getting this error when trying to run the software:

usb_claim_interface error -6
Failed to open rtlsdr device #0.

There some times it’s worked perfectly without any problems.

help how to solve it will be appreciated :)

**Comment by szpajder** [comment]
Either your device is in use by another application or you haven't
blacklisted RTL DVB-T kernel modules.

**Reply by YarinT** [reply]
There is a way to check which application using the device?
And what is the right way to blacklist the kernel modules? I’ve already tried multiple times without success.
Thanks
